Contrib/METU-NCC/MAT100/Ch7/7.5_cos2x_1.pg

```diff
@@ -0,0 +1,80 @@
+## BEGIN_DESCRIPTION
+##  trig eqn with cos2x
+## END_DESCRIPTION
+##
+##############################################
+## DBsubject(Trigonometry)
+## DBchapter(Analytic trigonometry)
+## DBsection(Solving trigonometric equations exactly)
+## Institution(METU-NCC)
+## Author(Benjamin Walter)
+## Level(2)
+## Language(en)
+## MO(1)
+###############################################
+
+
+DOCUMENT();        # This should be the first executable line in the problem.
+
+loadMacros(
+  "PGstandard.pl",
+  "PGcourse.pl",
+  "MathObjects.pl",
+);
+
+
+TEXT(beginproblem());
+
+COMMENT("one of eight different problems, three solutions");
+
+Context("Numeric");
+#Context()->constants->remove(pi);
+Context()->variables->add(pi=>['Real',TeX=>"\pi"]);
```

Comment:
I imagine that your aim here is to avoid allowing students to enter decimal approximations for the solutions. You can accomplish the same thing without adding pi as a variable:
```
Parser::Number::NoDecimals();
```